examples/camera-app/camera-common/include/camera-device-interface.h

```diff
@@ -207,6 +208,9 @@ class CameraDeviceInterface
         // Get the current camera viewport.
         virtual const ViewportStruct & GetViewport() = 0;
 
+        // Set the viewport for a specific stream
+        virtual CameraError SetViewport(VideoStream & stream, const ViewportStruct & viewPort) = 0;
```

Comment:
The expectation that the callee updates `stream` should be spelled out.

Reply:
Added a more complete comment header to the API.